homeassistant/components/automation/template.py

```diff
@@ -31,6 +31,6 @@ def template_listener(entity_id, from_s, to_s):
                 'from_state': from_s,
                 'to_state': to_s,
             },
-        }, context=to_s.context))
+        }, context=(to_s.context if to_s else None)))
```

Comment:
We should not set `None` to `context`. I have a open PR to fix similar issue, #21801